### DISCUSSION TITLE
Split window - two separate chats in one place

### DISCUSSION THREAD

**Discussion by InvaderJ** [danny-avila/LibreChat#5081] [Feature Requests & Suggestions]
Been trying to find an answer to this ahead of installing LibreChat, and have come up empty, so I'm going to assume this feature does not exist currently.

I really, really, could use a split window option. Basically two (or more) independent conversations side by side. This would be a huge efficiency booster for my workflows. 

This is a feature that Msty has and what I'm using currently, but LibreChat has a number of other features that make it more attractive. I've love to switch but the lack of being able to have at least two conversations going in a single window is a blocker.

Thanks!

**Comment by danny-avila** [comment]
It’s already supported but it’s a bit experimental and needs more refinement. Anyway, you can already try it today, type the Plus symbol “+” into the chat input to add an additional provider/model to chat within one window

**Reply by danny-avila** [reply]
<img width="414" alt="image" src="https://github.com/user-attachments/assets/4bfba6b9-960c-4ccf-908c-0663d1d1ede6" />

Here’s a screenshot (Only have access to a mobile device atm)

**Reply by InvaderJ** [reply]
Thanks, but that's not what I'm looking for. I'm not interested in sending one message to multiple models. I'm interested in literally having two or more conversations up, side by side. E.g. In one pane I am having a conversation about finance with Claude Sonnet 3.5, and then _right next to it_ in another pane, I am having another separate conversation, with the same model, about physics.  :)
